core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -54,7 +54,9 @@ object ApiVersion {
     "0.10.0" -> KAFKA_0_10_0_IV1,
 
     // introduced for JoinGroup protocol change in KIP-62
-    "0.10.1-IV0" -> KAFKA_0_10_1_IV0
+    "0.10.1-IV0" -> KAFKA_0_10_1_IV0,
+    // introduced ListGroupRequest v1 in KIP-79
+    "0.10.1-IV1" -> KAFKA_0_10_1_IV1
```

Comment:
Since KIP-74 will probably go in at about the same time, maybe there won't be a need for another bump?